README.md

```diff
@@ -1,2 +1,2 @@
 # TestRep
-SlackTesting.
```

Comment:
This is webhook comment test.

Reply:
Another test.

Reply:
And another.

Reply:
Another test

Reply:
Yea, another

Reply:
Gah, Again!

Reply:
WTH?

Reply:
FTW

Reply:
Yo

Reply:
shiza

Reply:
Holy

Reply:
OK, now?

Reply:
Now we'll see if comments are correct.

Reply:
OH, I think you forgot a comma, Aaron.

Reply:
This would be a longer message, perhaps describing something of interest.

Reply:
This would be a longer message, perhaps describing something of interest.

Reply:
What ever. 

I'm not sure what to think.

Reply:
hey hey hey

Reply:
Whatever.

I'm not sure what to think.

Reply:
lkjasdf

asdf


asdf

asdfa

Reply:
more stuff here to look at. Do this and that.

OK

Reply:
Nice!.

The `flat_map` also works in this case, and I think it reads well with `product`...

```
[2] pry(main)> { even: [ 0, 2 ], odd: [ 1, 3 ] }.flat_map { |k,v| [k].product(v) }
=> [[:even, 0], [:even, 2], [:odd, 1], [:odd, 3]]
```

Reply:
Hey, this is the comment.

Reply:
It's another comment to consider.

Reply:
This should be another color.

```text
[2] pry(main)> { even: [ 0, 2 ], odd: [ 1, 3 ] }.flat_map { |k,v| [k].product(v) }
=> [[:even, 0], [:even, 2], [:odd, 1], [:odd, 3]]
```

Reply:
This is just a comment.

Reply:
And here is one more comment. Let's see how it looks.

Reply:
will this work?

Reply:
Booya, this is what I think we need.

Reply:
And again?

Reply:
A Create Test.

Reply:
A Create Test.

Reply:
is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.

Reply:
cat

Reply:
is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.

Reply:
is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.

Reply:
Looks good!

Reply:
```ruby
puts "have you considered this?"
```

Reply:
What?